shareon/language/pt/acp/info_acp_shareon.php

```diff
@@ -0,0 +1,66 @@
+<?php
+/**
+*
+* Share On extension for the phpBB Forum Software package.
+* Brazilian Portuguese translation
+*
+* @copyright (c) 2015 Vinny <https://github.com/vinny>
+* @license GNU General Public License, version 2 (GPL-2.0)
+*
+*/
+
+/**
+* DO NOT CHANGE
+*/
+if (!defined('IN_PHPBB'))
+{
+	exit;
+}
+
+if (empty($lang) || !is_array($lang))
+{
+	$lang = array();
+}
+
+// DEVELOPERS PLEASE NOTE
+//
+// All language files should use UTF-8 as their encoding and the files must not contain a BOM.
+//
+// Placeholders can now contain order information, e.g. instead of
+// 'Page %s of %s' you can (and should) write 'Page %1$s of %2$s', this allows
+// translators to re-order the output of data while ensuring it remains correct
+//
+// You do not need this where single placeholders are used, e.g. 'Message %d' is fine
+// equally where a string contains only two placeholders which are used to wrap text
+// in a url you again do not need to specify an order e.g., 'Click %sHERE%s' is fine
+//
+// Some characters you may want to copy&paste:
+// ’ » “ ” …
+//
+
+$lang = array_merge($lang, array(
+	// ACP
+	'SO_ACP'		=> 'Share On Extension',
+	'SO_TITLE'		=> 'Configurações Share On Extension',
+	'SO_EXPLAIN'	=> 'Selecione em quais redes sociais deseja partilhar os seus tópicos.',	
+	'SHARE_ON_MOD'	=> 'Share On',
+	'SO_CONFIG'		=> 'Configurações',
+	'SO_SAVED'		=> 'Configurações alteradas com sucesso.',
+	'SO_SELECT'		=> 'Partilhar Em',
+	'SO_STATUS'		=> 'Ativar Share On Extension',
```

Comment:
It's not activar its ativar (there is a diference between Novo Acordo and Pré-Acordo, right now in portugal we write accordingly with the Novo Acordo)